napari/layers/utils/layer_utils.py

```diff
@@ -632,7 +632,7 @@ def dims_displayed_world_to_layer(
     return dims_displayed
 
 
-def get_extent_world(data_extent, data_to_world, centered=False):
+def get_extent_world(data_extent, data_to_world):
```

Comment:
This is technically part of the public API and we're removing a parameter name. As this is a utility, my guess is no-one is using this and this is OK, but just wanted to flag it in case other think otherwise.

Reply:
I added a warning; does that look reasonable?

Reply:
More than good enough for me.